openvdb/openvdb/unittest/TestDepth.cc

```diff
@@ -0,0 +1,74 @@
+// Copyright Contributors to the OpenVDB Project
```

Comment:
it's great that you add a unit-test for this change, but instead of created an entirely new file please add these tests to the existing TestValueAccessor.cc file